xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -22,6 +22,9 @@
 #endif
 #include "powermanagement/PowerManager.h"
 #include "profiles/ProfileManager.h"
+#if defined(TARGET_WINDOWS)
```

Comment:
I am no cpp developer nor do I know the Kodi architecture decent enough to judge but something tells me this is wrong. Shouldn't this be accessed from rendermanager?

Reply:
Moved to platform/win32/Win32Util.h

Reply:
I guess it still keeps the same problem, you're basically using windows specific includes in common classes. This needs to be abstracted somewhere (rendermanager?) and implemented in platform specific classes. I don't see this getting merged without this change.